test/cpu_fusion.cpp

```diff
@@ -2076,3 +2077,43 @@ TEST(cpu_fusion, sigmoid_multiply_fusion_backward)
                                                  expected_1);
     }
 }
+
+TEST(cpu_fusion, fuse_rnn_across_layer)
+{
+    pass::Manager pass_manager;
+    pass_manager.register_pass<runtime::cpu::pass::LSTMFusion>();
+    pass_manager.register_pass<runtime::cpu::pass::RNNFusion>();
+    pass_manager.register_pass<ngraph::pass::AlgebraicSimplification>();
+    pass_manager.register_pass<runtime::cpu::pass::RecurrentRNNFusion>();
+    const string json_path =
+        file_util::path_join(SERIALIZED_ZOO, "mxnet/2rnn_layer_1timestep.json");
+    const string json_string = file_util::read_file_to_string(json_path);
+    stringstream ss(json_string);
+    shared_ptr<Function> func = ngraph::deserialize(ss);
+    pass_manager.run_passes(func);
+    size_t ref_rnn_count = 1;
+    auto rnn_count = count_ops_of_type<op::Rnn>(func);
+    EXPECT_EQ(ref_rnn_count, rnn_count);
+}
+
+TEST(cpu_fusion, fuse_rnn_across_2layer_1timestep)
```

Comment:
Do you have a test with multiple layer and multiple time steps?

Reply:
i dont have the serialized graph where the feature size is same for Xt and ht for multiple time steps (which is the requirement for fusing across layer).  I hacked mxnet unit test to get the  current serialized graph. but for some reason i am not able to unroll those cells in the serialized graph. So each RNN layer have just 1 cell. I will try to get that.